### PR TITLE
Add referrerpolicy attribute on link elements

### PR DESCRIPTION
**Pull request by estark37**
Addresses issue #15 

@jeisinger PTAL?

### CODE REVIEW

File: index.src.html [hunk 1]
```diff
@@ -650,6 +655,17 @@ <h4 id="html-iframe-element-attributes">Attributes</h4>
...
+  <h3 id="html-link-element">HTMLLinkElement</h3>
+  <pre class="idl">
+    partial interface HTMLLinkElement {
+      attribute DOMString referrerPolicy;
+    };
+  </pre>
+  <h4 id="html-link-element-attributes">Attributes</h4>
+  The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
+  <code>referrerpolicy</code> content attribute, <a>limited to only known
+  values</a>.
```
Comment (jeisinger) [inline index.src.html]: I wonder whether we need to say something about how the referrerPolicy attribute is supposed to be applied here, or whether that's ok to be left open for interpretation?

Comment (estark37) [inline index.src.html]: Hmm... we don't say anything for the other elements, do we? I feel like it's pretty obvious how the attribute is applied, unless there's some `<link>` nuance that I don't know about (which would not be surprising).

Comment (annevk) [inline index.src.html]: Ideally it's defined in detail. E.g., how it's passed to Fetch and such. That's definitely part of the requirement to upstream this to HTML.

Comment (jeisinger) [inline index.src.html]: well, fetch doesn't talk about how it fetches css, or favicons, or what happens in the bookmark system, or how prerendering would work...

Comment (annevk) [inline index.src.html]: HTML defines how favicons and CSS are fetched. CSS should define how its subresources are fetched. Bookmarks navigate, which HTML defines and use fetch. I don't know what specification defines prerendering.

Comment (jeisinger) [inline index.src.html]: would you expect a bookmark to remember the referrer for its future navigations?

anyway, I guess the best we can do is saying something like if this link elements triggers a fetch or a navigation that creates a referrer from the current document, the referrer policy should apply.

Comment (annevk) [inline index.src.html]: I would expect bookmarks to always navigate without a referrer, but that's mostly up to the user agent since it's a UI feature.

Comment (estark37) [inline index.src.html]: Sorry for the delay. I still don't really get what's special about the link element that it needs special treatment here. The integration with Fetch is already defined by using Section 8.3 "Determine request's Referrer", which includes "If request was initiated by an element with a referrerpolicy content attribute..."

Comment (jeisinger) [inline index.src.html]: yeah, this PR LGTM

If there are issues with our general fetch integration, let's address them separately

**Comment by foolip** [comment]
Looks like https://w3c.github.io/webappsec-referrer-policy/ isn't automatically in sync with master? Any guess about when it'll include the `HTMLLinkElement` extensions?

**Comment by jeisinger** [comment]
no, we have to manually push to gh-pages... let me quickly do that

**Comment by foolip** [comment]
Thanks! https://w3c.github.io/webappsec-referrer-policy/#html-link-element

**Pull request merged**